tests/test_files/yaml_source_dir/valid_yamls_with_single_crd/dynatrace-monitoring.v0.2.0.clusterserviceversion.yaml

```diff
@@ -38,7 +38,8 @@ metadata:
   name: dynatrace-monitoring.v0.2.0
   namespace: "placeholder"
 spec:
-  apiservicedefinitions: {}
+  apiservicedefinitions:
+    owned: {}
```

Comment:
Are we requiring that if the `apiservicedefinitions` or `customresourcedefinitions` fields exist that they have `owned` as a sub field (or the presence of any valid sub field)?
(Also this should be a list not an object)

Reply:
Is our stance that if it exists and is empty that is valid? If so, these both should be valid:
```
spec:
  apiservicedefinitions: {}
```
and
```
spec:
  apiservicedefinitions:
    owned: []
```

Reply:
I want to clarify that my stance is based on my observations of the existing community-operators CSVs, many of which set `spec.apiservicedefinitions` to the empty object `{}`.
We *could* enforce that the key `owned` exists, but should we/does it create any benefit?

Reply:
Thanks. This was a mistake based on requiring `owned`. `owned` is not required, and this change is not needed.